heliopy/data/cassini.py

```diff
@@ -75,6 +77,27 @@ def mag_1min(starttime, endtime, coords):
         raise ValueError('coords must be one of {}'.format(valid_coords))
     base_url = ('http://pds-ppi.igpp.ucla.edu/ditdos/download?'
                 'id=pds://PPI/CO-E_SW_J_S-MAG-4-SUMM-1MINAVG-V1.0/DATA')
+    units = None
```

Comment:
I don't think you need this line.

Reply:
I added this since the program was giving an error that 'units' was referenced in the util.process before being initialized. Removed it now since the conditions work well to initialize the units!